lib/rubocop.rb

```diff
@@ -587,10 +587,12 @@
 require_relative 'rubocop/config_loader_resolver'
 require_relative 'rubocop/config_loader'
 require_relative 'rubocop/config_store'
+require_relative 'rubocop/config_validator'
 require_relative 'rubocop/target_finder'
 require_relative 'rubocop/token'
 require_relative 'rubocop/comment_config'
 require_relative 'rubocop/magic_comment'
+require_relative 'rubocop/obsolete_config'
```

Comment:
Maybe we should name this `config_something` for consistency with the other config-related classes - e.g. `config_obsoletions` or `config_changes` or something much better? :-)

Reply:
Good idea.